packages/genesys-spark-components/src/components/stable/gux-listbox/options/gux-option/gux-option.tsx

```diff
@@ -60,14 +82,18 @@ export class GuxOption {
           'gux-active': this.active,
           'gux-disabled': this.disabled,
           'gux-filtered': this.filtered,
-          'gux-selected': this.selected
+          'gux-selected': this.selected,
+          'gux-show-subtext': this.hasSubtext
         }}
         aria-selected={this.getAriaSelected()}
         aria-disabled={this.disabled.toString()}
       >
-        <gux-truncate ref={el => (this.truncateElement = el)}>
-          <slot />
-        </gux-truncate>
+        <div class="gux-option-wrapper">
+          <gux-truncate ref={el => (this.truncateElement = el)}>
+            <slot />
+          </gux-truncate>
+          <slot onSlotchange={() => this.onSubtextChange()} name="subtext" />
```

Comment:
So I noticed the other gux-option types don't have this onSlotchange. Should they have this logic as well?

Reply:
Updated